posts/2022-09-22-Rust-1.64.0.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: post
+title: "Announcing Rust 1.64.0"
+author: Josh Triplett
+team: The Release Team <https://www.rust-lang.org/governance/teams/release>
```

Comment:
Nit: we usually just sign these posts as the release team regardless of who writes them.